package/tilem/package

```diff
@@ -5,23 +5,37 @@
 pkgnames=(tilem)
 pkgdesc="TI-84+ calculator emulator"
 url=https://github.com/timower/rM2-stuff/tree/master/apps/tilem
-pkgver=0.0.7-3
-timestamp=2021-04-30T10:42Z
+_tag=0.1.1
```

Comment:
It would be somewhat more deterministic to depend on a commit sha instead of a tag, in case the tag ever gets deleted & recreated. 0.1.1 corresponds to `a383c22ff2ea7f4abe7aaaf936ed055a34f8523b` in case you're interested in making that change!

Reply:
While this is true, we've been happy to use tags as a standard throughout toltec, As they are supposed to be deterministic and not moved if you are using git the way it's meant to be used. So far, I haven't noticed timower moving tags, so I'm not too concerned.

Using tags also makes it easier to update the package, as you just need to update the version number without looking, the hash, and the timestamp.

Reply:
For sure. It does feel inconsistent to check hashes for fetching a tarball over HTTPS, but not a git fetch... but if this is already how many packages work, this is not the time / place to shake that out!

Reply:
That is a fair point. I'd be happy to approve and merge a new PR to switch this over to a commit hash if you'd like. I had forgotten that this was doing a git checkout.